llvm/test/Transforms/LoopVectorize/AArch64/clamped-trip-count.ll

```diff
@@ -21,3 +21,34 @@ for.body:                                         ; preds = %entry, %for.body
 for.cond.cleanup:                                 ; preds = %for.body
   ret void
 }
+
+define void @wide_tc_8(ptr nocapture %dst, i32 %n, i64 %val){
```

Comment:
nit: Perhaps this is better named as `@clamped_tc_max_8` to be consistent with the test name above?

Reply:
Done